server/prisma/seed.ts

```diff
@@ -0,0 +1,25 @@
+import { PrismaClient } from '@prisma/client'
+const prisma = new PrismaClient()
+
+async function main() {
+    const permissions = ['Product', 'Activity', 'Staff', 'Customer', 'Entry', 'Form']
+    for (var permission of permissions) {
+      const upsertPermissions = await prisma.feature.upsert({
+        where: { name: permission },
+        update: {},
+        create: {
+          name: permission
+        }
+      })
+      console.log(upsertPermissions)
```

Comment:
Not sure if this log is necessary

Reply:
this was for debugging purposes to verify it worked, will remove